### ISSUE TITLE
🐛 [Bug Report] TBC_DB : GOSSIP_OPTION_BANKER

### ISSUE THREAD

**Issue by Ischtaratu** [cmangos/issues#3107]
### Bug Details

TBC DB and deDE client
Darnassus
Bank NPC: Garryeth <Banker>
Bank NPC: Laim <Banker>
Same with Banker NPC in Ironforge and Shattrath (Aldor and Scryers)
When i click the NPC to open the Bank of Charakter (Not GuildBank) there is a Textstring: **GOSSIP_OPTION_BANKER** in NPC window

So there is something wrong with textstrings, maybe missing localised string?

### Steps to Reproduce

1. right click on npc
2.

### Expected behavior

Other Text

### Suggested Workaround

not needed, you can open Bankaccount, but the NPC Text is wrong

### Core SHA1 Commit Hash

Development Build 06.08.22 from github

### Database SHA1 Commit Hash

Development Build 06.08.22 from github

### Operating System

W10 x64 21H2

### Client Version

2.4.3 (The Burning Crusade)

**Comment by NeatElves** [comment]
> UPDATE creature_template SET NpcFlags=NpcFlags&~1 WHERE Entry=**ID_BANKER**;

**Comment by MantisLord** [comment]
I cannot seem to reproduce this one. I talked to Garryeth and Lairn as well as the other bankers mentioned and I am always taken straight to the bank screen - no gossip/text is displayed for me.
Tested on both TBC and WotLK and the result is the same.

**Comment by Ischtaratu** [comment]
![grafik](https://user-images.githubusercontent.com/6404203/194883131-db0b101c-9f97-406d-930a-6308ca122ccf.png)

Garryeth (ID 4209) is the middle Banker in Darnassus. DB is a clean development Build from today (10.10.2022)
Tested in 2 different Languages ... so no translation issue

**Comment by AnonXS** [comment]
Player Bot xD

**Comment by Ischtaratu** [comment]
> Player Bot xD

i never be alone :)

By the way, I have found that such a bot group is best played as a tank. Then you can set the targets and have the DD's and the healer attached. Leading the group as a magician or healer is also possible, but sometimes has its disadvantages.

**Comment by AnonXS** [comment]
this gossip option seems to only be triggered due to the forced gossip with playerbot, hence this is why nobody could confirm this.

**Comment by Ischtaratu** [comment]
That makes perfect sense. But the question is why not all bank NPCs are affected. After a deeper research I discovered that among the BankNPCs altogether 3 different flags were assigned. Some are obviously related to other quests. I have now changed the flags of the affected NPC's. With the result that of course the NPC bot option has disappeared. So far understandable. But even if I may be the only player with bot. Wouldn't it make more sense to remove the bot option from all NPC's and insert an optional additional NPC that you only have to place in the main cities?
